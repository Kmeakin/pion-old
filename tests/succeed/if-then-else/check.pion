//~ run = "${PION} elab ${FILE}"

(if true then 1 else 0) : Int
