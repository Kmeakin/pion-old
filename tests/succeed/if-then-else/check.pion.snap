exit status: 0
stdout:
match true {
    false => 0,
    true => 1,
} : Int

stderr:
